include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -373,7 +373,8 @@ typedef enum OrtAllocatorType {
 } OrtAllocatorType;
 
 /**
-   memory types for allocator, exec provider specific types should be extended in each provider
+ * memory types for allocator, exec provider specific types should be extended in each provider
+ * Whenever this struct is updated, please also update the MakeKey function in onnxruntime/core/framework/execution_provider.cc
 */
```

Comment:
Needs a comment either here or in execution_provider.cc saying what update should be done to the MakeKey function as that's clear.

Reply:
Added